src/__tests__/to-be-disabled.tsx

```diff
@@ -49,6 +49,20 @@ describe('.toBeDisabled', () => {
       expect(() => expect(queryByTestId(name)).not.toBeDisabled()).toThrow();
     });
   });
+
+  test('handle editable prop for TextInput', () => {
+    const { getByTestId } = render(
+      <View>
+        <TextInput testID="disabled" editable={false} />
+        <TextInput testID="enabled-by-default" />
+        <TextInput testID="enabled" editable />
+      </View>,
+    );
+
+    expect(getByTestId('disabled')).toBeDisabled();
```

Comment:
Pls update in a similar way as suggested in `toBeEnabled` comment by adding `getByPlaceholder` queries.